### PR TITLE
[stable/zfs-localpv]: update charts to 1.4.0

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR

- update chart version
- update README
- update values.yaml

### CODE REVIEW

File: deploy/helm/charts/README.md [hunk 1]
```diff
@@ -93,20 +93,20 @@ The following table lists the configurable parameters of the OpenEBS ZFS Localpv
...
+| `zfsController.snapshotController.image.registry`| Registry for snapshot-controller image| `k8s.gcr.io/`|
+| `zfsController.snapshotController.image.repository`| Image repository for snapshot-controller| `sig-storage/snapshot-controller`|
 | `zfsController.snapshotController.image.pullPolicy`| Image pull policy for snapshot-controller| `IfNotPresent`|
 | `zfsController.snapshotController.image.tag`| Image tag for snapshot-controller| `v2.0.1`|
-| `zfsController.provisioner.image.registry`| Registry for csi-provisioner image| `quay.io/`|
-| `zfsController.provisioner.image.repository`| Image repository for csi-provisioner| `k8scsi/csi-provisioner`|
+| `zfsController.provisioner.image.registry`| Registry for csi-provisioner image| `k8s.gcr.io/`|
+| `zfsController.provisioner.image.repository`| Image repository for csi-provisioner| `sig-storage/csi-provisioner`|
 | `zfsController.provisioner.image.pullPolicy`| Image pull policy for csi-provisioner| `IfNotPresent`|
 | `zfsController.provisioner.image.tag`| Image tag for csi-provisioner| `v1.6.0`|
```
Comment (prateekpandey14) [inline deploy/helm/charts/README.md:111]: nit : v2.1.0

File: deploy/helm/charts/README.md [hunk 2]
```diff
@@ -93,20 +93,20 @@ The following table lists the configurable parameters of the OpenEBS ZFS Localpv
...
+| `zfsController.resizer.image.registry`| Registry for csi-resizer image| `k8s.gcr.io/`|
+| `zfsController.resizer.image.repository`| Image repository for csi-resizer| `sig-storage/csi-resizer`|
 | `zfsController.resizer.image.pullPolicy`| Image pull policy for csi-resizer| `IfNotPresent`|
 | `zfsController.resizer.image.tag`| Image tag for csi-resizer| `v0.4.0`|
-| `zfsController.snapshotter.image.registry`| Registry for csi-snapshotter image| `quay.io/`|
-| `zfsController.snapshotter.image.repository`| Image repository for csi-snapshotter| `k8scsi/csi-snapshotter`|
+| `zfsController.snapshotter.image.registry`| Registry for csi-snapshotter image| `k8s.gcr.io/`|
+| `zfsController.snapshotter.image.repository`| Image repository for csi-snapshotter| `sig-storage/csi-snapshotter`|
 | `zfsController.snapshotter.image.pullPolicy`| Image pull policy for csi-snapshotter| `IfNotPresent`|
 | `zfsController.snapshotter.image.tag`| Image tag for csi-snapshotter| `v2.0.1`|
```
Comment (prateekpandey14) [inline deploy/helm/charts/README.md:103]: v4.0.0

File: deploy/helm/charts/README.md [hunk 3]
```diff
@@ -93,20 +93,20 @@ The following table lists the configurable parameters of the OpenEBS ZFS Localpv
 | `zfsNode.nodeSelector`| Nodeselector for zfsnode daemonset pods| `""`|
 | `zfsNode.tolerations` | zfsnode daemonset's pod toleration values | `""`|
 | `zfsNode.securityContext` | Seurity context for zfsnode daemonset container | `""`|
-| `zfsController.resizer.image.registry`| Registry for csi-resizer image| `quay.io/`|
-| `zfsController.resizer.image.repository`| Image repository for csi-resizer| `k8scsi/csi-resizer`|
+| `zfsController.resizer.image.registry`| Registry for csi-resizer image| `k8s.gcr.io/`|
+| `zfsController.resizer.image.repository`| Image repository for csi-resizer| `sig-storage/csi-resizer`|
 | `zfsController.resizer.image.pullPolicy`| Image pull policy for csi-resizer| `IfNotPresent`|
 | `zfsController.resizer.image.tag`| Image tag for csi-resizer| `v0.4.0`|
```
Comment (prateekpandey14) [inline deploy/helm/charts/README.md:99]: v1.1.0

File: deploy/helm/charts/README.md [hunk 4]
```diff
@@ -93,22 +93,22 @@ The following table lists the configurable parameters of the OpenEBS ZFS Localpv
...
+| `zfsController.snapshotter.image.repository`| Image repository for csi-snapshotter| `sig-storage/csi-snapshotter`|
 | `zfsController.snapshotter.image.pullPolicy`| Image pull policy for csi-snapshotter| `IfNotPresent`|
-| `zfsController.snapshotter.image.tag`| Image tag for csi-snapshotter| `v2.0.1`|
-| `zfsController.snapshotController.image.registry`| Registry for snapshot-controller image| `quay.io/`|
-| `zfsController.snapshotController.image.repository`| Image repository for snapshot-controller| `k8scsi/snapshot-controller`|
+| `zfsController.snapshotter.image.tag`| Image tag for csi-snapshotter| `v4.0.0`|
+| `zfsController.snapshotController.image.registry`| Registry for snapshot-controller image| `k8s.gcr.io/`|
+| `zfsController.snapshotController.image.repository`| Image repository for snapshot-controller| `sig-storage/snapshot-controller`|
 | `zfsController.snapshotController.image.pullPolicy`| Image pull policy for snapshot-controller| `IfNotPresent`|
 | `zfsController.snapshotController.image.tag`| Image tag for snapshot-controller| `v2.0.1`|
```
Comment (prateekpandey14) [inline deploy/helm/charts/README.md:107]:
```suggestion
| `zfsController.snapshotController.image.tag`| Image tag for snapshot-controller| `v4.0.0`|
```

**Comment by pawanpraka1** [comment]
@shubham14bajpai can you update the all the crds and csi-provisioner args also. see this https://github.com/openebs/zfs-localpv/pull/271/files.

**Comment by shubham14bajpai** [comment]
> @shubham14bajpai can you update the all the crds and csi-provisioner args also. see this https://github.com/openebs/zfs-localpv/pull/271/files.

The crds are already in v1 version [refer](https://github.com/openebs/zfs-localpv/blob/36e0f69fd001dd1864d046890a32b1b38eb29fe9/deploy/helm/charts/crds/volumesnapshotclasses.yaml#L3) and the csi-provisioner args are also up to date [refer](https://github.com/openebs/zfs-localpv/blob/36e0f69fd001dd1864d046890a32b1b38eb29fe9/deploy/helm/charts/templates/zfs-contoller.yaml#L74). Pushed a missing rbac and will rebase after #290 gets merged.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**